apps/ocsp.c

```diff
@@ -53,6 +53,18 @@ NON_EMPTY_TRANSLATION_UNIT
 #  define LOG_ERR       2
 # endif
 
+# if defined(OPENSSL_SYS_VXWORKS)
+/* setpgid() & fork() are not supported */
```

Comment:
I'd prefer if you change the comment to  `/* setpgid() and fork() are not supported */`, or even better, make it two comments before the respective functions:  (`/* setpgid() is not supported */` resp. `/* fork() is not supported */`, or simply `/* not supported */` for both).